src/metric_model_reporter.cc

```diff
@@ -178,12 +186,13 @@ void
 MetricModelReporter::InitializeCounters(
     const std::map<std::string, std::string>& labels)
 {
-  // Always setup these counters, regardless of config
+  // Always setup these inference request metrics, regardless of config
   counter_families_["inf_success"] = &Metrics::FamilyInferenceSuccess();
   counter_families_["inf_failure"] = &Metrics::FamilyInferenceFailure();
   counter_families_["inf_count"] = &Metrics::FamilyInferenceCount();
   counter_families_["inf_exec_count"] =
       &Metrics::FamilyInferenceExecutionCount();
+  gauge_families_[kPendingRequestMetric] = &Metrics::FamilyInferenceQueueSize();
```

Comment:
Separate initialize function?

Reply:
Good catch, I must've been too tired when writing this :sweat_smile: